src/java.base/share/classes/jdk/internal/misc/CDS.java

```diff
@@ -25,7 +25,26 @@
 
 package jdk.internal.misc;
 
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import jdk.internal.access.JavaLangInvokeAccess;
+import jdk.internal.access.SharedSecrets;
+
 public class CDS {
+    /**
+      * indicator for dumping class list.
+      */
+    static public final boolean isDumpingClassList;
```

Comment:
what about making this a private static field and adding a public static `isDumpingClassList()` method (which was in the previous version).

Reply:
That will have a name for two properties, if you that is OK, I will use the previous version.